### DISCUSSION TITLE
Showing markers and geojson layers at a height above the plane

### DISCUSSION THREAD

**Discussion by pietervdvn** [maplibre/maplibre-gl-js#2441] [Q&A]
Hi,

Is there a way to show that a marker is actually on a height?

For example, [this AED](https://pietervdvn.github.io/mc/feature/maplibre/aed.html?z=17.8&lat=51.20822&lon=3.225021&language=nl&layer-range=false#node/8738768519) is actually located on the 7th floor. It would thus be nice that, if the map is pinched, the marker would show on the 7th floor as well.

**Comment by HarelM** [comment]
See the following proposal, feel free to better define it so this can move forward.
https://github.com/maplibre/maplibre-style-spec/issues/62
